### ISSUE TITLE
A&R -> Ability for Rec Services/ Admin to add sub areas

### ISSUE THREAD

**Issue by Dianadec** [bcgov/bcparks-ar-admin#353]
#### Description:

Context:
Might have many new sub areas coming up (traffic counters discovered) - Access to south coast data map. 49 counters installed in south coast. So how to incorporate them into the system?

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by meyerdarcie** [comment]
@sallyhaynes - what activity will traffic counters fall under? Are they for day use?

**Comment by sallyhaynes** [comment]
@meyerdarcie Yes, they will all be under day use, and are a mix of traffic and trail counters (the majority being trail). So it would affect data under "vehicle count" and "trail count". 
![image](https://github.com/user-attachments/assets/35b83afa-c8a3-4387-94d1-a0362e20bcda)

**Comment by LindsayMacfarlane** [comment]
October 11, 2024 Prioritization session - discussion notes:

- Regions have said they don't have time to enter the data and information, which would mean it would fall to Sally and Nicole to administer and they have limited capacity.
- This would help automate the process
- Can start with South Coast as a pilot.

**Comment by Dianadec** [comment]
Todo:
- designers work with devs and to see what the design could be - feedback from admin users
- get some concrete user needs down prior to discussion.
- so, need to setup session. Create a ticket to do the exploratory work

Deadlines?
- depends on the level of complexity of this ticket and how much work it would be
- Season launch happens in April, so if we want to incorporate these for next season (again, not a hard deadline), we would need to have it done before then.

**Comment by nicolebottoms** [comment]
this one ties into #333 for the notes field to be added for the park, i dont want it to get lost!

**Comment by LindsayMacfarlane** [comment]
January 22, 2025: Identified as a high priority. [Mural board notes and discussion](https://app.mural.co/t/bcparks2575/m/bcparks2575/1728427370222/4c9714df914c7ec8bdcf20a9ee1999cc9e664a27?wid=4-1737568561245).
